cmd/goal/application.go

```diff
@@ -1369,36 +1369,29 @@ var methodAppCmd = &cobra.Command{
 				return
 			}
 
-			// specify the return hash prefix
-			hashRet := sha512.Sum512_256([]byte("return"))
-			hashRetPrefix := hashRet[:4]
-
-			var abiEncodedRet []byte
-			foundRet := false
-			if resp.Logs != nil {
-				for i := len(*resp.Logs) - 1; i >= 0; i-- {
-					retLog := (*resp.Logs)[i]
-					if bytes.HasPrefix(retLog, hashRetPrefix) {
-						abiEncodedRet = retLog[4:]
-						foundRet = true
-						break
-					}
-				}
+			// the 4-byte prefix for logged return values
+			var abiReturnHash = []byte{0x15, 0x1f, 0x7c, 0x75}
```

Comment:
Why the change from calculating it?

Reply:
I guess we decided that the return format is first 4 byte of hash of "return", and we wrote the 4 bytes down in each SDK implementations, so we don't have to recalculate it again.

I think it is okay to define the pre-calculated prefix, anything against that?

Reply:
The SDKs all hardcode this value, so I didn't see a need to calculate it fresh every time.

Reply:
Just added a comment explaining where this values comes from